src/Compilers/CSharp/Portable/Parser/Lexer_StringLiteral.cs

```diff
@@ -263,13 +283,16 @@ public InterpolatedStringScanner(Lexer lexer)
             {
                 _lexer = lexer;
 
-                _isVerbatim = (lexer.TextWindow.PeekChar(0) == '$' && lexer.TextWindow.PeekChar(1) == '@') ||
-                              (lexer.TextWindow.PeekChar(0) == '@' && lexer.TextWindow.PeekChar(1) == '$');
+                _kind =
+                    (lexer.TextWindow.PeekChar(0) == '$' && lexer.TextWindow.PeekChar(1) == '@') ||
```

Comment:
nit: if you feel the inclination it might be nice to write this as something like

`(lexer.TextWindow.PeekChar(0), lexer.TextWindow.PeekChar(1)) is ('$', '@') or ('@', '$')`

Reply:
i like that!